lib/realtime/tenants/janitor.ex

```diff
@@ -65,7 +65,7 @@ defmodule Realtime.Tenants.Janitor do
     Logger.info("Janitor started")
     %{chunks: chunks, tasks: tasks} = state
     all_tenants = :ets.select(@table_name, [{{:"$1"}, [], [:"$1"]}])
-    connected_tenants = :ets.select(@syn_table, [{{:"$1", :_, :_, :_, :_, :_}, [], [:"$1"]}])
+    connected_tenants = :ets.select(@syn_table, [{{:"$1", :_, :_, :_, :_, Node.self()}, [], [:"$1"]}])
```

Comment:
Sweet match specs. Technically it probably should be `{:const, Node.self()}` but I think that it will not cause any issues there.

Reply:
will do the change so it's more correct

Reply:
<img width="1443" alt="Screenshot 2025-02-20 at 11 02 38" src="https://github.com/user-attachments/assets/e183092f-3828-4aee-9c30-00c7e037ced4" />
yeah this is better and more correct :D